recipes/r-boutroslabplottinggeneral/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: boutroslabplottinggeneral
+  version: "5.3.4"
+
+source:
+  fn: BoutrosLab.plotting.general_5.3.4.tar.gz
+  url: http://bpg.oicr.on.ca/src/contrib/BoutrosLab.plotting.general_5.3.4.tar.gz
+
+requirements:
+  build:
+      - r-base
+      - r-rcolorbrewer
+      - r-lattice
+      - r-latticeextra
+      - r-r.methodss3
+      - r-hexbin
+      - r-proto
+      - r-findpython
+      - r-getopt
+      - r-rjson
+      - r-argparse
+      - r-assertthat
+      - r-r6
+      - r-rcpp
+      - r-lazyeval
+      - r-tibble
+      - r-magrittr
+      - r-dbi
+      - r-bh
+      - r-dplyr
+
+  run:
+      - r-base
```

Comment:
I was overthinking it. Test and dependencies added. Build works on my end and test seems to succeed. 

+ source /home/rmorin/anaconda3/bin/activate /home/rmorin/anaconda3/conda-bld/r-boutroslabplottinggeneral_1486335682971/_t_env
+ /bin/bash -x -e /home/rmorin/anaconda3/conda-bld/r-boutroslabplottinggeneral_1486335682971/test_tmp/run_test.sh
+ /home/rmorin/anaconda3/conda-bld/r-boutroslabplottinggeneral_1486335682971/_t_env/bin/R -e 'library('\''BoutrosLab.plotting.general'\'')'

blah.blah.blah...

TEST END: boutroslabplottinggeneral-5.3.4-r3.3.1_0